netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -158,11 +158,30 @@ typedef struct _net_ebpf_extension_sock_addr_wfp_filter_context
     bool v4_attach_type;
 } net_ebpf_extension_sock_addr_wfp_filter_context_t;
 
+static uint64_t
+_ebpf_sock_addr_context_create(
+    _In_reads_bytes_opt_(data_size_in) const uint8_t* data_in,
+    _In_ size_t data_size_in,
+    _In_reads_bytes_opt_(context_size_in) const uint8_t* context_in,
+    _In_ size_t context_size_in,
```

Comment:
```suggestion
    size_t context_size_in,
```
Don't put `_In_` on value parameters, only pointer parameters.  Same on line 164, 1342, etc.